dev-util/xmake/xmake-2.8.5.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="A cross-platform build utility based on Lua"
+DESCRIPTION="What is Xmake? \
+1. Xmake is a cross-platform build utility based on the Lua scripting language. \
+2. Xmake is very lightweight and has no dependencies outside of the standard library. \
+3. Uses the \`xmake.lua\` file to maintain project builds with a simple and readable syntax. \
+Xmake can be used to directly build source code (like with Make or Ninja), or it can generate project source files like CMake or Meson. It also has a built-in package management system to help users integrate C/C++ dependencies."
+HOMEPAGE="https://xmake.io/"
+COPYRIGHT="2015-present TBOOX Open Source Group"
```

Comment:
I don't think having `present` in a copyright is terribly useful.  Should probably be just the start year of `2015` or the full date range of `2015-2023`

Reply:
Changed it to 2023 for now - the present is what the package actually uses (and what's on their homepage)

Reply:
That's just so they don't have to remember to keep updating it every year.  Copyrights are generally assumed to be valid from whatever year they started and extend into the future.

In any case, I don't feel strongly about it, I just think `present` is silly and I don't recall any other recipes doing that.